src/app/containers/MediaPlayer/Canonical.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { string } from 'prop-types';
+import styled from 'styled-components';
+import Image from '@bbc/psammead-image';
+
+const StyledPlaceholder = styled.div`
+  cursor: pointer;
+  position: absolute;
```

Comment:
Could be wrong but the way the image placeholder works is that it has `position: relative;`, and the elements inside of the placeholder have `position: absolute;` - this differs from that approach

Reply:
`StyledContainer` is what I would define as the placeholder, so I'm curious what is the difference between `StyledContainer` and `StyledPlaceholder`

Reply:
The `StyledContainer` is the parent component that is sized to either 16:9 or 9:16. It needs a `position: relative` so that its child can be positioned `absolute`.

The `StyledPlaceholder` is the wrapper around the placeholder image. It adds a `cursor: pointer` to indicate it is a clickable element, and is sized to fill its parent container (the `StyledContainer`).

Reply:
Spoke IRL. The container and placeholder seem to me that they could be a single element rather than 2 different ones. Also IMO the `StyledContainer` is acting as the placeholder to the video component, rather than a container - this naming could be reviewed